packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1554,15 +1583,107 @@ class ConstrainedBox extends SingleChildRenderObjectWidget {
   }
 }
 
+/// A container that imposes no constraints on its child, allowing it to render
```

Comment:
I'd probably avoid the word container here too. It's not a [Container], per se. Maybe just "a widget..."

Reply:
Done.